Update README.md

smasato opened the pull request:
This PR adds information to README.md and organizes it.

Is there any other information I should add?

mame commented:
Great, thank you! Now that a new command `typeprof --init` has been proposed (#310) and I will merge it soon. It would be better to write that procedure instead of manually putting `typeprof.conf.json`.

smasato commented:
@mame 
I replaced the procedure about configuration files with the use of `typeprof --init`.
I also added a link to the marketplace for the VS Code extension.
